shenyu-plugin/shenyu-plugin-sign/src/main/java/org/apache/shenyu/plugin/sign/service/DefaultSignService.java

```diff
@@ -126,6 +126,6 @@ private Pair<Boolean, String> sign(final ShenyuContext shenyuContext, final Serv
     }
 
     private String buildExtSignKey(final String signKey, final ShenyuContext shenyuContext) {
-        return String.join("", Constants.TIMESTAMP, shenyuContext.getTimestamp(), Constants.PATH, shenyuContext.getPath(), Constants.VERSION, "1.0.0", signKey);
+        return String.join("", Constants.PATH, shenyuContext.getPath(), Constants.TIMESTAMP, shenyuContext.getTimestamp(), Constants.VERSION, "1.0.0", signKey);
```

Comment:
Why change this order? According to the documentation?

Reply:
![image](https://user-images.githubusercontent.com/42602026/201695269-ea672d8f-c2c4-4aeb-911e-8461f46631db.png)

after https://github.com/apache/shenyu/pull/4089 ,Actually its funticon doesnt meet the documentation description,I change that ,it can be compatible  when there is no request body. but  its still incompatible ,when there have request body

Reply:
![image](https://user-images.githubusercontent.com/42602026/206338819-8f6cb2dd-83b4-4315-a313-6835ff70ee92.png)



In #4242, I changed it back according to the current(2.5.1) document . The comments above are based on the 2.5.0 documentation